<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>431</width>
      <height>491</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="bookPage">
     <widget class="QGroupBox" name="roomSizeBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>111</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Room Size</string>
      </property>
      <widget class="QRadioButton" name="sizeQueenButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>2 Queen Bed</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="sizeKingButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>1 King Bed</string>
       </property>
      </widget>
     </widget>
     <widget class="QPlainTextEdit" name="nameTextField">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="roomViewBox">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>70</y>
        <width>111</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>View</string>
      </property>
      <widget class="QRadioButton" name="viewStandardButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Standard</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="viewAtriumButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Atrium</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>261</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string>Guests</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Adults</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="adultSpinbox">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>4</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="childSpinbox">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>50</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>3</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>47</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Children</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>210</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Parking</string>
      </property>
      <widget class="QRadioButton" name="parkingButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>1 Vehicle</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>300</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="title">
       <string>Cost</string>
      </property>
      <widget class="QLabel" name="priceLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>$0.00</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>60</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Check-In</string>
      </property>
      <widget class="QDateEdit" name="checkInDate">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimumDateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2020</year>
         <month>5</month>
         <day>5</day>
        </datetime>
       </property>
       <property name="date">
        <date>
         <year>2020</year>
         <month>5</month>
         <day>5</day>
        </date>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="bookButton">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>450</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Book</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>130</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Nights</string>
      </property>
      <widget class="QSpinBox" name="nightsSpinbox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>7</number>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="roomPreviewLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>271</width>
        <height>191</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="checkoutPage">
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>141</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>Total</string>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>40</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Room: </string>
       </property>
      </widget>
      <widget class="QLabel" name="roomCostLabel">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>30</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>$0.00</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>40</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Parking:</string>
       </property>
      </widget>
      <widget class="QLabel" name="parkingFeeLabel">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>60</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>$0.00</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>40</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Resort:</string>
       </property>
      </widget>
      <widget class="QLabel" name="resortFeeLabel">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>90</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>$0.00</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>40</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Total:</string>
       </property>
      </widget>
      <widget class="QLabel" name="totalLabel">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>120</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>$0.00</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>221</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>Payment Method</string>
      </property>
      <widget class="QTextEdit" name="creditCardTextField">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Credit Card Number</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="expDate">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>80</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
       <property name="date">
        <date>
         <year>2020</year>
         <month>5</month>
         <day>5</day>
        </date>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Expiration</string>
       </property>
      </widget>
      <widget class="QLabel" name="errorTextField">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>171</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="cardTypeLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>171</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="checkoutButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>450</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Checkout</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="confirmPage">
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>301</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>Reservation Details</string>
      </property>
      <widget class="QDateEdit" name="checkInDateConfirm">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="date">
        <date>
         <year>2020</year>
         <month>5</month>
         <day>5</day>
        </date>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Check-In Date</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nights</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Parking:</string>
       </property>
      </widget>
      <widget class="QLabel" name="parkingLabel">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>80</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>No</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Adults</string>
       </property>
      </widget>
      <widget class="QLabel" name="adultsLabelConfirm">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>100</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QLabel" name="childrenLabelConfirm">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>120</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Children</string>
       </property>
      </widget>
      <widget class="QLabel" name="nightsConfirm">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>50</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_9">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>180</y>
        <width>301</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Payment Information</string>
      </property>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Card:</string>
       </property>
      </widget>
      <widget class="QLabel" name="cardConfirmLabel">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>xxxx</string>
       </property>
      </widget>
      <widget class="QLabel" name="confirmCharge">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>40</y>
         <width>47</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>$0.00</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Total Charged:</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="confirmButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>450</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="agreementBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>450</y>
        <width>221</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>You Will Agree To Have a Wonderful Stay</string>
      </property>
     </widget>
     <widget class="QLabel" name="discountLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>300</y>
        <width>431</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
